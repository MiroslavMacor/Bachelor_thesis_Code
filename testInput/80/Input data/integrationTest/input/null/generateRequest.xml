<?xml version="1.0"?>
<remsig>
<person>
<userId>359</userId>
</person>
<password></password>
</remsig>
