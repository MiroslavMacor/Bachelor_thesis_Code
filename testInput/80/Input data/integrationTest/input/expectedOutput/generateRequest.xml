<?xml version="1.0" encoding="UTF-8"?><remsig>  <requestId>38</requestId>  <certificateRequest>-----BEGIN CERTIFICATE REQUEST-----MIICVjCCAT4CAQAwETEPMA0GA1UEAxMGUmVtU2lnMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAw+/Txx/mMuJpKNQwp6HlPxp2zThed3DKLOyv1mfGIm70VLl3hYDkEBRcz8BlbGxFDpoyhBfwGDM7teM7kgz4PCX9E2XF7lZG8r3KcYl4BPPfEZFW/lEntvh7auCJfgYwS5a+2b2D6Jr6/uHbplo/CE4ryCYaMssQsntDtSOaPrMSGkzcTmiSjUPHjPuaC6ewB8aXxSG3+c2scXaN0QR+Qw1/6H2hKjTOOE7sennGtlrSwJsc53oeOKjyzKOOzhgmzpDB8AYfBrEZVfcB7CJuf8YW+Qduw4ySGeoZgv9TAdF0jnBrA093RS45ik2GB/TrI6f+hkDWEkPtPTJhxMtSoQIDAQABoAAwDQYJKoZIhvcNAQELBQADggEBAMH3JfKQUTZmAI8bj/LXeoZOYH2d6d3vOInUF15dp3lrZ087dXYF8XB0MTczfusLfvPgepTDgQ/Lfj+Kjh5wYNgsUxOfiUr03z5AXyGHHSDgitQ19NUVQ6nfi7/ZfX+5bZ6bq1kd2V0IuThS5A+7xiQp11zd2xOssLGVLBr/2rOcsm2DsdFN/MObZXcKC+KqgHyjP2l1XqreONxbfUyM41sQ1KXQQ0ilkXrAt3A5g1nUp3aDTvIjxdKKGqqC+ex3vgNtEu42jP0j/6FIRimokJrnqqebjU4RddCE+ESpwAYV0o3FvVZrLr+FPUzVLvd24UoeXmcP16Q/tB16q17miHs=-----END CERTIFICATE REQUEST-----</certificateRequest>  <subjectKey>3a32adc5be31bc7cd2d4e2dc743f8526e0609051</subjectKey>  <operationId>1463326316485</operationId></remsig>
